testing/src/main/scala/sbt/JUnitXmlTestsListener.scala

```diff
@@ -33,9 +33,10 @@ import sbt.protocol.testing.TestResult
  * report format.
  * @param outputDir path to the dir in which a folder with results is generated
  */
-class JUnitXmlTestsListener(val outputDir: String, logger: Logger) extends TestsListener {
+class JUnitXmlTestsListener(val outputDir: String, logger: Logger, legacyTestReport: Boolean)
```

Comment:
Could you move the `legacyTestReport: Boolean` to second position, and also supply an alternative constructor the the previous signature for backward binary compatibility please?

```
[error]  * method this(java.lang.String,sbt.util.Logger)Unit in class sbt.JUnitXmlTestsListener does not have a correspondent in current version
```